quantecon_book_networks/plotting.py

```diff
@@ -47,9 +47,6 @@ def plot_graph(A,
                 edge_colors.append(node_color_list[i])
                 width = a * edge_size_multiple
                 edge_widths.append(width)
-                
-    # Get rid of self-loops
-    G.remove_edges_from(nx.selfloop_edges(G))         
```

Comment:
@HengchengZhang have self loops been removed from the original data?

Reply:
Yes. Using this API in `networkx` to remove self-loops causes our difference in figures between codebook and pdf version of the book.

I now remove the self-loops in codebook by setting the diagonal of matrix to zero.

Reply:
OK I am not sure I fully understand why the `api` would do that -- it should effectively do the same thing as setting the diagonal of an `adjacency` matrix to zero.

Reply:
I agree that they should behave the same but I didn't research further in this case. 

Only thing I know for now is that the figures are generated correctly if I manually set them to zeros and vice versa.

My guess for the reason is probably because of the order of applying color lists and generating plots.